### PR TITLE
Add schemas for feature manifests

### PR DESCRIPTION
**Pull request by brennie**
Because

- the feature manifest schema is used in Experimenter and desktop;
- there are bugs in the schema (e.g., the definition of hasExposure);
  and
- the field names in the Python definition are not pythonic

This commit

- adds a copy of the schema from experimenter/features to
  mozilla_nimbus_schemas;
- updates the field names with aliases to be more pythonic; and
- adds a unit test to validate all vendored manifests.

### CODE REVIEW

File: schemas/mozilla_nimbus_schemas/experiments/feature_manifests.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    STRING = "string"
+    BOOLEAN = "boolean"
+    JSON = "json"
+
+
+FEATURE_SCHEMA_TYPES = {
+    FeatureVariableType.INT: "integer",
+    FeatureVariableType.STRING: "string",
+    FeatureVariableType.BOOLEAN: "boolean",
+}
```
Comment (mikewilli) [inline schemas/mozilla_nimbus_schemas/experiments/feature_manifests.py:18]: What do these do?

**Review by mikewilli — CHANGES_REQUESTED** [review]

File: schemas/mozilla_nimbus_schemas/tests/experiments/fixtures/feature_manifests/firefox-desktop.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+"no-feature-firefox-desktop":
```
[the remark this answers is not in the document]
Comment (brennie) [reply schemas/mozilla_nimbus_schemas/tests/experiments/fixtures/feature_manifests/firefox-desktop.yaml:1]: Apparently it doesn't. I just copied & pasted it from desktop.

File: schemas/mozilla_nimbus_schemas/experiments/feature_manifests.py (hunk 1 above)
Comment (brennie) [reply schemas/mozilla_nimbus_schemas/experiments/feature_manifests.py:18]: Oops, don't need these. These were copy+pasted from the other python module.

File: schemas/mozilla_nimbus_schemas/experiments/feature_manifests.py [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    FeatureVariableType.BOOLEAN: bool,
+}
+
+
+class FeatureVariable(BaseModel):
+    description: Optional[str]
+    enum: Optional[list[str]]
+    fallback_pref: Optional[str] = Field(alias="fallbackPref")
+    type: Optional[FeatureVariableType]
+    set_pref: Optional[str] = Field(alias="set_pref")
```
[the remark this answers is not in the document]
Comment (brennie) [reply schemas/mozilla_nimbus_schemas/experiments/feature_manifests.py:32]: yes

File: schemas/mozilla_nimbus_schemas/tests/experiments/test_feature_manifests.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+import pytest
+import yaml
+
+from mozilla_nimbus_schemas.experiments import (
+    FeatureManifest,
+    FirefoxDesktopFeatureManifest,
+)
+
+FIXTURE_DIR = Path(__file__).parent / "fixtures" / "feature_manifests"
+REPO_ROOT_DIR = Path(__file__).parent.parent.parent.parent.parent
```
[the remark this answers is not in the document]
Comment (brennie) [reply schemas/mozilla_nimbus_schemas/tests/experiments/test_feature_manifests.py:12]: Yeah it should be fine. I can remove this test though.

**Review by mikewilli — APPROVED** [review]
Looks good to me!

**Pull request merged**